aries-site/src/layouts/main/Layout.js

```diff
@@ -1,91 +1,28 @@
 import React from 'react';
-import Head from 'next/head';
+
 import PropTypes from 'prop-types';
-import { Box, Grommet, Main, ResponsiveContext } from 'grommet';
-import { AnchorGroup, Nav } from 'aries-core';
+import { Grommet } from 'grommet';
 
 import { aries } from '../../themes/aries';
-import { SideBar, SideBarItemList } from '../navigation';
-
-const filterChildren = (children, type) => {
-  const filteredChildren = React.Children.map(children, child => {
-    return child.props[type] ? child : null;
-  });
-
-  if (filteredChildren.length > 1) {
-    console.warn(
-      `Expected a single ${type}, received ${filteredChildren.length}.`,
-      `Only first ${type} element will be rendered.`,
-    );
-  }
-
-  return filteredChildren;
-};
-
-export const Layout = ({ children, title }) => {
-  // Only expect a single child of the following types
-  const mainContent = filterChildren(children, 'MainContent');
-
-  // TODO selectedNav should be retrived from aries-core
-  // as the selected element of the NavBar
-  const selectedNav = 'start';
+import { Header, Head, NavLayout } from '..';
 
+export const Layout = ({ children, title, isLanding }) => {
```

Comment:
Suggestion: rename `isLanding` to `showLinks`, keep the same functionality, but is direct to the functionality and less concerned about how it is being used.

Reply:
The context of `showLinks` is too specific for line #14.